app/routes/tasks.js

```diff
@@ -5,11 +5,21 @@ const API_BASE_URL = ENV.BASE_API_URL;
 
 export default class TasksRoute extends Route {
   model = async () => {
-    const response = await fetch(`${API_BASE_URL}/tasks/self`, {
+    await fetch(`${API_BASE_URL}/tasks/self`, {
       credentials: 'include',
-    });
-    const allTasks = await response.json();
-
-    return allTasks;
+    })
+      .then(function (response) {
+        if (!response.ok) {
+          throw Error(response.statusText);
```

Comment:
Response not ok, means any case other than 200, right?
What about 401? That's the user's fault that they're not logged in
Also, what will you do by throwing these errors on frontend? How will the user know? 😅

Let's not rely on response.ok and stuff
If response code is 401, show an alert saying please log in to continue and redirect them to the realdevsquad.com homepage